content/zh/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -50,44 +61,49 @@ To set feature gates for a component, such as kubelet, use the `--feature-gates`
 The following tables are a summary of the feature gates that you can set on
 different Kubernetes components.
 -->
-下表总结了可以在不同的 Kubernetes 组件上设置的功能开关。
+
+下表总结了在不同的 Kubernetes 组件上可以设置的功能开关。
 
 <!--
 - The "Since" column contains the Kubernetes release when a feature is introduced
   or its release stage is changed.
 - The "Until" column, if not empty, contains the last Kubernetes release in which
   you can still use a feature gate.
 - If a feature is in the Alpha or Beta state, you can find the feature listed
-  in the Alpha/Beta feature gate table.
-- If a feature is stable (GA) or deprecated, you can find all stages for that feature listed
-  in the GA/Deprecated feature gate table.
+  in the [Alpha/Beta feature gate table](#feature-gates-for-alpha-or-beta-features).
+- If a feature is stable you can find all stages for that feature listed in the
+  [Graduated/Deprecated feature gate table](#feature-gates-for-graduated-or-deprecated-features).
+- The [Graduated/Deprecated feature gate table](#feature-gates-for-graduated-or-deprecated-features)
+  also lists deprecated and withdrawn features.
 -->
+
 - 引入功能或更改其发布阶段后，"Since" 列将包含 Kubernetes 版本。
 - "Until" 列（如果不为空）包含最后一个 Kubernetes 版本，您仍可以在其中使用功能开关。
-- 如果某个功能处于 Alpha 或 Beta 状态，则可以在 Alpha 或者 Beta 功能开关表中找到该功能。
-- 如果某个功能稳定（GA）或已弃用，则可以在 GA 或者不推荐使用的功能开关表中找到该功能的所有阶段。
+- 如果某个功能处于 Alpha 或 Beta 状态，您可以在 [Alpha 和 Beta 功能开关表](#feature-gates-for-alpha-or-beta-features)中找到该功能。
```

Comment:
> Please check all links like this.

@gavinfish  Yes, it is invalid, but I don't think that modifying only one page will solve the problem.

It is an anchor that points to a title in the current page. In fact, almost all the anchors of the [doc/zh](https://kubernetes.io/zh/docs/home/) pages are invalid. 

The current [specification](https://github.com/k8smeetup/k8s-official-translation#%E7%BF%BB%E8%AF%91%E8%A7%84%E8%8C%83) of the k8smeetup community explicitly requires that the English anchor be retained (even if it will cause an anchor failure), so I kept the English ahchor just like [everyone else](https://github.com/kubernetes/website/pulls?page=2&q=is%3Apr+label%3Alanguage%2Fzh+is%3Aclosed+label%3Algtm).

Reply:
Thanks for clarifying @gorda